src/transaction/combine.ts

```diff
@@ -1,43 +1,66 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+/**
+ * The transactions should all be equal EXCEPT for the 'Signers' field. 
```

Comment:
i don't think "except" needs to be all-caps in this case. the all-caps implies that a caller could do something wrong if they don't do whatever the all-caps indicates, but in this case this is just what the function does, NBD to user.

Reply:
if you're in here already, "equivalence" would be even more descriptive than "equality".